### ISSUE TITLE
[Bug]diffrent party bugs

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#2019]
im at 181h2

https://gyazo.com/223ca68bd39035576140294ff3f4ebaf

party share not working
when kill monster credit count is wrong. the one who killed the monster get credited twice.

in latest version 182c2
party is more bugged, there you not even can see the party members in list.

**Comment by vgabler** [comment]
Was about to file an issue too. Facing the same problems.
In v1.82 the members list is OK, but exp / item sharing is not working.

when exp / item sharing is OFF, everything works as usual. When it's ON, no one gets any reward from killing monsters.

**Comment by insthync** [comment]
I can't reproduce party members list issues yet, the list is showing properly.

I found EXP sharing issues, but the item sharing is fine, going to fix it

**Comment by insthync** [comment]
@vgabler EXP sharing issues fixed by 

swap parameters when calling `GivingRewardToParty` function in `BaseMonsterCharacterEntity` class, from
```
GivingRewardToParty(tempPlayerCharacterEntity, isLastAttacker, reward, shareGuildExpRate, rewardRate, makeMostDamage, out givenRewardExp, out givenRewardCurrency);
```
to 
```
GivingRewardToParty(tempPlayerCharacterEntity, isLastAttacker, reward, rewardRate, shareGuildExpRate, makeMostDamage, out givenRewardExp, out givenRewardCurrency);
```

**Comment by insthync** [comment]
@moepi2k 

![Screenshot_447](https://user-images.githubusercontent.com/2697550/232337433-6bb74694-50dc-4dce-ae6c-f8aa39467e96.png)

```
if (isLastAttacker && playerCharacterEntity.ObjectId != nearbyPartyMember.ObjectId)
```

**Comment by insthync** [comment]
Fixed in 1.82c3
